sdk_container/src/third_party/coreos-overlay/sys-kernel/coreos-modules/files/commonconfig-6.1

```diff
@@ -885,6 +885,7 @@ CONFIG_TCP_CONG_HYBLA=m
 # CONFIG_TCP_CONG_WESTWOOD is not set
 CONFIG_TCP_MD5SIG=y
 CONFIG_TIGON3=m
+CONFIG_TLS=y
```

Comment:
>  By integrating the TLS module, applications and containers will have the ability to offload TLS processing to the kernel.

That's not what this config option does. How did you validate that this has the desired outcome? Are you able to test TLS offload?

Reply:
I meant symmetric encryption handling of the TLS protocol can be done in-kernel @jepio

Reply:
Oh ok, my bad - i misunderstood the intent here.

So someone actually wants to TLS software offload to the kernel. In that case make it a module, so that we don't increase the size the default kernel size for something rarely used.